### ISSUE TITLE
not possible to hide 'Remember Me' checkbox

### ISSUE THREAD

**Issue by kupe3** [CakeDC/users#410]
in the TemplateUsers/login.ctp, line 26, there is
```
        if (Configure::check('Users.RememberMe.active')) {
            ....
        }
```
and regardless of the value of the Users.RememberMe.active, the checkbox is displayed. I think that should be changed to 
```
        if (Configure::read('Users.RememberMe.active')) {
            ....
        }
```
so if i set 
```
Configure::write('Users.RememberMe.active',false);
```
they 'remember me' checkbox is not visible and probably this functionality is disabled

**Comment by ajibarra** [comment]
Closed by #418

**Comment by kupe3** [comment]
i upgraded the plugin to dev-master and i see the change implmented. However on clean plugin install if i just hide the 'remember me' checkbox on the form and i try to login i get following error:

`Fatal error: Call to a member function read() on a non-object in D:\web_server\anita\vendor\cakedc\users\src\Auth\RememberMeAuthenticate.php on line 36`
